jordgubb/snake_ll.sy

```diff
@@ -0,0 +1,259 @@
+from love use love
+use gfx
+use sfx
+use input
+
+MaybeSegment :: enum
+    Just Segment,
+    None,
+end
+
+Segment :: blob {
+    position: (int, int),
+    child: *,
```

Comment:
:(

Reply:
It is not optimal. I don't know if it is worth fixing before the jam...